lua/autorun/swamp_core.lua

```diff
@@ -57,7 +57,9 @@ function math.power2(n)
     return math.pow(2, math.ceil(math.log(n) / math.log(2)))
 end
 
--- TABLE
+BLACK = Color(0, 0, 0, 255)
```

Comment:
there shouldn't be my changes in this PR, i think you need to fetch and merge upstream/master then push to origin/master